django/river/settings/base.py

```diff
@@ -206,7 +206,7 @@
 # CorsHeaders
 # Used to access api from third-party domain
 
-CORS_URLS_REGEX = r"^/api/.*$"
+# CORS_URLS_REGEX = r"^/api/.*$"
```

Comment:
?

Reply:
Before cors policies were only applied on host/api but since the oidc routes are on host/oidc, it didn't worked

@vmttn should I remove this or let it commented ?